.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to cs102
```

Comment:
`CONTRIBUTING.md` is typically in the repository root, not hidden in `.github`. I find it better to advertise the file as much as possible, whereas things like the PR/issue templates are fine being hidden away in a directory.